semopenalex-app/data/templates/http%3A%2F%2Fwww.metaphactory.semopenalex.com%2Fapp%2FAbout.html

```diff
@@ -1,34 +1,48 @@
-          <li>A publicly accessible <a href='/sparql' target='_blank'>SPARQL endpoint</a> containing the latest SemOpenAlex data.</li>
-          <li><a href='https://doi.org/10.5281/zenodo.7912776' target='_blank'>Entity Embeddings</a> for all works, authors, sources and institutions in SemOpenAlex.</li>
+          <li>A publicly accessible <a href='/sparql' target='_blank'>SPARQL endpoint</a> containing the latest
+            SemOpenAlex data.</li>
+          <li><a href='https://doi.org/10.5281/zenodo.7912776' target='_blank'>Entity Embeddings</a> for all works,
+            authors, sources and institutions in SemOpenAlex.</li>
         </ol>
       </div>
     </div>
 
-     <div style='margin:auto;width:922px;padding-top:16px;'>
+    <div style='margin:auto;width:922px;padding-top:16px;'>
       <b>How big is SemOpenAlex?</b>
       <div style='margin-top:8px;'>
         <bs-row>
           <bs-col md="6">
-            {{>::count-entity type='https://semopenalex.org/ontology/Work' image='/assets/images/icon-2.png' label='works'}}
-            {{>::count-entity type='https://semopenalex.org/ontology/Author' image='/assets/images/icon-1.png' label='authors'}}
-            {{>::count-entity type='https://semopenalex.org/ontology/Institution' image='/assets/images/icon-3.png' label='institutions'}}
+            {{>::count-entity type='https://semopenalex.org/ontology/Work' image='/assets/images/icon-2.png'
+            label='works'}}
+            {{>::count-entity type='https://semopenalex.org/ontology/Author' image='/assets/images/icon-1.png'
+            label='authors'}}
+            {{>::count-entity type='https://semopenalex.org/ontology/Institution' image='/assets/images/icon-3.png'
+            label='institutions'}}
+            {{>::count-entity type='https://semopenalex.org/ontology/Publisher' image='/assets/images/icon-7.png'
+            label='publishers'}}
+            {{>::count-entity type='https://semopenalex.org/ontology/Funder' image='/assets/images/funding.png'
+            label='funders'}}
           </bs-col>
           <bs-col md="6">
-            {{>::count-entity type='http://www.w3.org/2004/02/skos/core#Concept' image='/assets/images/icon-5.png' label='topics'}}
-            {{>::count-entity type='https://semopenalex.org/ontology/Source' image='/assets/images/icon-6.png' label='sources'}}
-            {{>::count-entity type='https://semopenalex.org/ontology/Publisher' image='/assets/images/icon-7.png' label='publishers'}}
-            {{>::count-entity type='https://semopenalex.org/ontology/Funder' image='/assets/images/funding.png' label='funders'}}
+            {{>::count-entity type='https://semopenalex.org/ontology/Source' image='/assets/images/icon-6.png'
+            label='sources'}}
+            {{>::count-entity type='http://www.w3.org/2004/02/skos/core#Concept' image='/assets/images/icon-5.png'
+            label='concepts'}}
```

Comment:
images with `icon-x` are deleted. please use proper image name. you can see in the main search page.